recipe/meta.yaml

```diff
@@ -46,3 +88,4 @@ about:
 extra:
   recipe-maintainers:
     - jakirkham
+    - jjhelmus
```

Comment:
As I absorbed your PR, @jjhelmus, you are added as a maintainer. Hope this is still ok. Please let me know either way.